### PR TITLE
Refactor address join fields to a different function

### PR DESCRIPTION
**Pull request by cesarlr**
Hi!
@robinkeunen we need this refactor to localize address field easily and not rewriting the whole `share_subscription` method.
It can help to split this method in smaller concrete methods than can be inherited.

### CODE REVIEW

File: easy_my_coop_website/controllers/main.py [hunk 1]
```diff
@@ -383,6 +383,12 @@ def get_share_product(self, share_product_id, **kw):
             }
         }
 
+    def format_address(self, house_number, street_name):
```
Comment (robinkeunen) [inline easy_my_coop_website/controllers/main.py]: I'd mark this function private : `def _format_address(...)`.

**Review by robinkeunen — APPROVED** [review]
LGTM 

thanks for contributing :-)

**Review by enricostano — APPROVED** [review]

**Comment by cesarlr** [comment]
Hi! Can it be merged?

**Comment by remytms** [comment]
@cesarlr In fact we are facing bugs with this functionality (not your dev, but de split in the address field). It works fine for the first time a cooperator request a share, but if it already owns a share and that the address is not formatted well nothing appear in the address field.

Splitting address is not a trivial dev, as address can have many format, even in the same country. So we decided to roll back and use the Odoo standard for addresses. #167 

This roll back will be merged and deployed today.

If you need to tweak address field, I suggest you to put that in a separated module.

I close this PR, but feel free to reopen it if needed. We can also continue to discuss about addresses in this PR.

@robinkeunen FYI.

**Pull request closed without merging**